cli/tests/pcluster/models/cluster_dummy_model.py

```diff
@@ -237,3 +251,29 @@ def dummy_efs(mount_dir, name="name", file_system_id=None):
         provisioned_throughput=500,
         file_system_id=file_system_id,
     )
+
+
+class DummyClusterBucket(ClusterBucket):
```

Comment:
can we avoid creating Dummy subclasses? I'm just afraid that we end up writing unit tests that test Dummy code. Is it possible to instantiate a ClusterBucket with the necessary mocking?

Reply:
remove subclass.